schema/deploy/shipping/views.sql

```diff
@@ -139,26 +139,37 @@ create or replace view shipping.return_results_v1 as
              when count(present) > 0 then 'complete'
            end as status,
            -- We only return the top level organisms for results so we want to omit subtypes
-           array_agg(distinct subpath(organism, 0, 1)::text)
-            filter (where present and organism is not null)  as organisms_present,
-           array_agg(distinct subpath(organism, 0, 1)::text)
-            filter (where not present and organism is not null) as organisms_absent
+           array_agg(distinct
+             (case
+              when 'Influenza'::ltree @> organism then subpath(organism, 0, 2)::text
+              else subpath(organism, 0, 1)::text
+             end)
+           ) filter (where present and organism is not null)  as organisms_present,
+           array_agg(distinct
+             (case
+              when 'Influenza'::ltree @> organism then subpath(organism, 0, 2)::text
+              else subpath(organism, 0, 1)::text
+             end)
+           ) filter (where not present and organism is not null)  as organisms_absent
 
       from warehouse.identifier
       join warehouse.identifier_set using (identifier_set_id)
       left join warehouse.sample on uuid::text = sample.collection_identifier
-      left join shipping.presence_absence_result_v1 on sample.identifier = presence_absence_result_v1.sample
+      left join shipping.presence_absence_result_v2 on sample.identifier = presence_absence_result_v2.sample
 
-    --These are all past and current collection identifier sets not including self-test
+    --These are all past and current collection identifier sets not including Y1 Flu@Home self-test
     where identifier_set.name in ('collections-seattleflu.org',
                                   'collections-kiosks',
                                   'collections-environmental',
                                   'collections-swab&send',
+                                  'collections-self-test',
                                   'collections-household-observation',
                                   'collections-household-intervention') and
           (organism is null or
           -- We only return results for these organisms, so omit all other presence/absence results
-          organism <@ '{"Adenovirus", "Human_coronavirus", "Enterovirus", "Influenza", "Human_metapneumovirus", "Human_parainfluenza", "Rhinovirus", "RSV"}'::ltree[])
+          organism <@ '{"Influenza.A", "Influenza.B", "RSV"}'::ltree[]) and
+          -- Limit to only Cepheid results
+          presence_absence_result_v2.details ->> 'device' = 'Cepheid'
```

Comment:
It's a small thing, but may make a difference: `details @> '{"device":"Cepheid"}'` can use the index on `details`, but `details->>'device' = 'Cepheid'` cannot. Details in the doc: https://www.postgresql.org/docs/10/datatype-json.html#JSON-INDEXING

Reply:
I believe this condition will also negate the `left join` to presence/absence results and filter out rows which would have status of `notReceived` or `processing`.